src/adc_readout/AdcSettings.cpp

```diff
@@ -31,6 +60,41 @@ void setCLIArguments(CLI::App &Parser, AdcSettings &ReadoutSettings) {
                   "Kafka broker.")
       ->group("ADC Readout Options")
       ->default_str("AdcDemonstrator");
+
+  auto ParseOffsetTimestamp =
+      [&ReadoutSettings](std::vector<std::string> const &Input) -> bool {
+    auto TestString = Input.at(0);
+    std::transform(TestString.begin(), TestString.end(), TestString.begin(),
```

Comment:
Is there not a library function that can do this. Does boost have to_lower, should we put something like 
this helper into src/common ?

Reply:
1. Weirdly, there is no function for doing this on a std::string in the standard library.
2. Yes, boost has a function that does this but I do not think it is worth it to add that dependency for this (relatively minor) functionality.
3. Yes, it *might* be worth it to add a function that does this into *common* but of course then we have the problems of: does the developers (i.e. us) know that it is there and how often do we use it assuming that we do remember.